GPU/Vulkan/DrawEngineVulkan.cpp

```diff
@@ -256,6 +241,18 @@ void DrawEngineVulkan::DestroyDeviceObjects() {
 	for (int i = 0; i < VulkanContext::MAX_INFLIGHT_FRAMES; i++) {
 		frame_[i].Destroy(vulkan);
 	}
+
+	if (pushVertex) {
```

Comment:
Hm, I think it would be good to set `pushUBO = nullptr;`.  Also might be a little better if these followed the `_` convention...

-[Unknown]